cdap-ui/app/cdap/components/404/index.js

```diff
@@ -48,7 +48,9 @@ export default function Page404({ entityName, entityType, children }) {
           <div className="navigation-section">
             <div>
               {T.translate(`${I18N_PREFIX}.subtitleMessage2`)}
-              <Link to={`/ns/${namespace}/`}>{T.translate(`${I18N_PREFIX}.overviewLabel`)}</Link>
+              <Link to={`/ns/${namespace}/control`}>
```

Comment:
i wonder if we should tell user to go to home instead of control center?

Reply:
@yukiej also had the same comment. @bdmogal What do you think we should do?

Reply:
Talked to Bhooshan, redirecting users to home page here.